### PR TITLE
GH-97850: Suppress cross-references to removed ``importlib.util`` functions

### PR DESCRIPTION
**Pull request by AA-Turner**
* Issue: gh-97850

----
:books: Documentation preview :books:: https://cpython-previews--104134.org.readthedocs.build/

### CODE REVIEW

File: Misc/NEWS.d/3.12.0a1.rst [hunk 1]
```diff
@@ -2028,8 +2028,8 @@ resources.
 .. nonce: NzdREm
 .. section: Library
 
-Remove deprecated :func:`importlib.utils.set_loader` and
-:func:`importlib.utils.module_for_loader` from :mod:`importlib.utils`.
+Remove deprecated :func:`!importlib.utils.set_loader` and
+:func:`!importlib.utils.module_for_loader` from :mod:`importlib.utils`.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.12.0a1.rst]: Seems like there's no such module -- let's fix it while we're here :)

```suggestion
Remove deprecated :func:`!importlib.util.set_loader` and
:func:`!importlib.util.module_for_loader` from :mod:`importlib.util`.
```

Comment (AA-Turner) [reply Misc/NEWS.d/3.12.0a1.rst]: Oops -- fell into autopilot! Thanks for catching

**Comment by AA-Turner** [comment]
There's one other instance of ``importlib.utils`` (Misc/NEWS.d/3.10.0a5.rst; line 502) -- is this worth picking up at all?

A

**Review by AlexWaygood — APPROVED** [review]

**Comment by AlexWaygood** [comment]
> There's one other instance of `importlib.utils` (Misc/NEWS.d/3.10.0a5.rst; line 502) -- is this worth picking up at all?
> 
> A

Oh yeah, let's fix that now :)

**Comment by AA-Turner** [comment]
Brill, done just now.

A

**Pull request merged**